src/m365/planner/commands/task/task-add.ts

```diff
@@ -105,6 +108,25 @@ class PlannerTaskAddCommand extends GraphCommand {
       }, (err: any): void => this.handleRejectedODataJsonPromise(err, logger, cb));
   }
 
+  private getPriorityValue(options: Options): number | undefined {
+    const priority = options.priority;
+    
+    if (typeof priority === "string") {
+      switch (priority.toLowerCase()) {
+        case "urgent":
+          return 1;
+        case "important":
+          return 3;
+        case "medium":
+          return 5;
+        case "low":
+          return 9;
+      }
+    }
+    
+    return priority as number | undefined;
+  }
```

Comment:
this method could be then also moved to the `taskPriority` (we should think of a better name for sure.. sorry for being not creative but I had only one coffee 😋)

Reply:
This may seem useful to me. But I don't have a better name. I'm not creative either. 😃